### PR TITLE
[script.module.addon.signals] 0.0.5+matrix.1

### PR DESCRIPTION
**Pull request by ruuk**
### Description

I _think_ I versioned this correctly :smile:.

Here's a link to the changes vs. gotham: 
https://github.com/ruuk/script.module.addon.signals/compare/558f50f7d7d15e56cfbf5838362281c7e70dff30...master-kodi19

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.module.addon.signals/.pylintrc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[MESSAGES CONTROL]
```
Comment (basrieter) [inline script.module.addon.signals/.pylintrc:1]: File does not need to be include in this PR

File: script.module.addon.signals/.travis.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+language: python
```
Comment (basrieter) [inline script.module.addon.signals/.travis.yml:1]: File does not need to be include in this PR

File: script.module.addon.signals/Makefile [hunk 3]
```diff
@@ -0,0 +1,58 @@
+export PYTHONPATH := $(CURDIR)/lib:$(CURDIR)/test
```
Comment (basrieter) [inline script.module.addon.signals/Makefile:1]: File does not need to be include in this PR

File: script.module.addon.signals/requirements.txt [hunk 4]
```diff
@@ -0,0 +1,4 @@
+kodi-addon-checker
```
Comment (basrieter) [inline script.module.addon.signals/requirements.txt:1]: File does not need to be include in this PR

File: script.module.addon.signals/setup.py [hunk 5]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/python
```
Comment (basrieter) [inline script.module.addon.signals/setup.py:1]: File does not need to be include in this PR

File: script.module.addon.signals/test/test_encoding.py [hunk 6]
```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
```
Comment (basrieter) [inline script.module.addon.signals/test/test_encoding.py:1]: Exclude the 'test' folder please.

File: script.module.addon.signals/test/xbmcaddon.py [hunk 7]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
```
Comment (basrieter) [inline script.module.addon.signals/test/xbmcaddon.py:1]: Exclude the 'test' folder please.

File: script.module.addon.signals/tox.ini [hunk 8]
```diff
@@ -0,0 +1,27 @@
+[tox]
```
Comment (basrieter) [inline script.module.addon.signals/tox.ini:1]: This file does not need to be include in this PR

File: script.module.addon.signals/addon.xml [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+<addon id="script.module.addon.signals"
+    name="Addon Signals"
+    version="0.0.5+matrix.1"
+    provider-name="Rick Phillips (ruuk)">
+    <requires>
+        <import addon="xbmc.python" version="3.0.0"/>
+    </requires>
+    <extension point="xbmc.python.module" library="lib" />
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en">Inter-addon signalling</summary>
```
Comment (enen92) [inline script.module.addon.signals/addon.xml:11]: en_gb

File: script.module.addon.signals/addon.xml [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+        <description lang="en">Provides signal/slot mechanism for inter-addon communication</description>
```
Comment (enen92) [inline script.module.addon.signals/addon.xml:12]: Same

File: script.module.addon.signals/changelog.txt [hunk 11]
```diff
@@ -0,0 +1,25 @@
+Changelog for Addon Signals
```
Comment (enen92) [inline script.module.addon.signals/changelog.txt:1]: File not needed

**Pull request merged**